src/components/Definitions/Definitions.scss

```diff
@@ -40,6 +40,7 @@ main {
   .status-icon {
     margin-right: 0.5em;
     vertical-align: sub;
+    float: left;
```

Comment:
I think this change should be added to pipelineRuns.scss under the packages dir, this causes the story book and third party consumes to still have the issue 

storybook screen grab with the changeset

![Screenshot 2019-12-13 at 14 58 26](https://user-images.githubusercontent.com/18128367/70809166-1e233300-1db9-11ea-84b8-f47adb42e966.png)

Reply:
ah ok, I'll update that now